### PR TITLE
Mapstruct unmapped

### PR DESCRIPTION
**Pull request by SpikeCyrill**

### CODE REVIEW

File: mapstruct-unmapped/pom.xml [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    </parent>
+    <modelVersion>4.0.0</modelVersion>
+
+    <artifactId>mapstruct-unmapped</artifactId>
+    <dependencies>
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-context</artifactId>
+            <version>${springframework.version}</version>
+        </dependency>
```
Comment (jeffzhu2010) [inline mapstruct-unmapped/pom.xml]: My comments

File: mapstruct/src/test/java/com/baeldung/mapper/unmapped/DocumentMapperWithConfigUnitTest.java [hunk 2]
```diff
@@ -0,0 +1,44 @@
+package com.baeldung.mapper.unmapped;
+
+import com.baeldung.dto.DocumentDTO;
+import com.baeldung.entity.Document;
+import org.junit.jupiter.api.Test;
+
+import java.util.Arrays;
+import java.util.Date;
+
+import static org.junit.Assert.assertEquals;
```
Comment (ashleyfrieze) [inline mapstruct/src/test/java/com/baeldung/mapper/unmapped/DocumentMapperWithConfigUnitTest.java]: You're mixing JUnit 5 tests with JUnit 4 assertions... I think that's a bit tricky to follow.

File: mapstruct/src/main/java/com/baeldung/entity/Document.java [hunk 3]
```diff
@@ -0,0 +1,15 @@
+package com.baeldung.entity;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.util.Date;
+
+@Getter
+@Setter
```
Comment (ashleyfrieze) [inline mapstruct/src/main/java/com/baeldung/entity/Document.java]: While I know what these lombok annotations are doing, it's probably easier not to mix Lombok into this article on another subject - explicit getters/setters is more transparent. Your IDE can generate them, and then the reader can see them.

File: mapstruct-unmapped-policy/src/test/java/com/baeldung/mapper/DocumentMapperUnitTest.java [hunk 4]
```diff
@@ -0,0 +1,46 @@
...
+        entity.setId(1);
+        entity.setTitle("Price 13-42");
+        entity.setText("List of positions.......");
+        entity.setModificationTime(new Date());
+
+        DocumentDTO dto = DocumentMapper.INSTANCE.documentToDocumentDTO(entity);
+
+        assertEquals(dto.getId(), entity.getId());
+        assertEquals(dto.getTitle(), entity.getTitle());
+        assertEquals(dto.getText(), entity.getText());
```
Comment (ashleyfrieze) [inline mapstruct-unmapped-policy/src/test/java/com/baeldung/mapper/DocumentMapperUnitTest.java]: I find it inconsistent to use JUnit 4 assertions with JUnit 5. Please can you switch to the `jupiter` assertions, or even try assertJ, which is much nicer - `assertThat(dto.getId()).isEqualTo(entity.getId());`

File: mapstruct/src/test/java/com/baeldung/mapper/unmapped/DocumentMapperMappingIgnoreUnitTest.java [hunk 5]
```diff
@@ -0,0 +1,44 @@
+package com.baeldung.mapper.unmapped;
+
+import com.baeldung.dto.DocumentDTO;
+import com.baeldung.entity.Document;
+import org.junit.Test;
+
+import java.util.Arrays;
+import java.util.Date;
+
+import static org.junit.Assert.assertEquals;
```
Comment (ashleyfrieze) [inline mapstruct/src/test/java/com/baeldung/mapper/unmapped/DocumentMapperMappingIgnoreUnitTest.java]: Another case of mixed JUnits...

**Pull request closed without merging**